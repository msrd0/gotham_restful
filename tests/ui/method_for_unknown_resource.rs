#[macro_use] extern crate gotham_restful;

#[read_all(UnknownResource)]
fn read_all()
{
}

fn main()
{
}
