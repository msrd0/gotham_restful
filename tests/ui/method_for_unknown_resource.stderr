error[E0412]: cannot find type `UnknownResource` in this scope
 --> $DIR/method_for_unknown_resource.rs:3:12
  |
3 | #[read_all(UnknownResource)]
  |            ^^^^^^^^^^^^^^^ not found in this scope
